tests/data/dataconverter/readers/mpes/config_file.json

```diff
@@ -14,328 +14,399 @@
-  "/ENTRY[entry]/DATA[data]/data": "@data:data",
-  "/ENTRY[entry]/DATA[data]/data/@units": "counts",
-  "/ENTRY[entry]/DATA[data]/AXISNAME[*]": "@data:*.data",
-  "/ENTRY[entry]/DATA[data]/AXISNAME[*]/@units": "@data:*.unit"
+  "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/COLLECTIONCOLUMN[collectioncolumn]": {
+    "projection": "@attrs:metadata/instrument/analyzer/projection",
+    "scheme": "angular dispersive",
+    "mode": "@attrs:metadata/instrument/analyzer/lens_mode",
+    "extractor_voltage": "@attrs:metadata/file/KTOF:Lens:Extr:V",
+    "extractor_voltage/@units": "V",
+    "extractor_current": "@attrs:metadata/file/KTOF:Lens:Extr:I",
+    "extractor_current/@units": "µA",
+    "working_distance": 4.0,
+    "working_distance/@units": "mm",
+    "LENS_EM[lens_*{A,B,C,D,E,F,G,H,I,UCA,UFA,Foc}]": {
+      "name": "*",
+      "voltage": "@attrs:metadata/file/KTOF:Lens:*:V",
+      "voltage/@units": "V"
+    },
+    "field_aperture": {
+      "shape": "@attrs:metadata/instrument/analyzer/fa_shape",
+      "size": "@attrs:metadata/instrument/analyzer/fa_size",
+      "size/@units": "µm",
+      "POSITIONER[fa_m1]": {
+        "value": "@attrs:metadata/file/KTOF:Apertures:m1.RBV",
+        "value/@units": "mm"
+      },
+      "POSITIONER[fa_m2]": {
+        "value": "@attrs:metadata/file/KTOF:Apertures:m2.RBV",
+        "value/@units": "mm"
+      }
+    },
+    "contrast_aperture": {
+      "shape": "@attrs:metadata/instrument/analyzer/ca_shape",
+      "size": "@attrs:metadata/instrument/analyzer/ca_size",
+      "size/@units": "µm",
+      "POSITIONER[ca_m3]": {
+        "value": "@attrs:metadata/file/KTOF:Apertures:m3.RBV",
+        "value/@units": "mm"
+      }
+    }
+  },
+  "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/ENERGYDISPERSION[energydispersion]": {
+    "pass_energy": "@attrs:metadata/file/KTOF:Lens:TOF:V",
+    "pass_energy/@units": "eV",
+    "scheme": "tof",
+    "tof_distance": 0.9,
+    "tof_distance/@units": "m"
+  },
+  "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/DETECTOR[detector]": {
+    "amplifier_type": "MCP",
+    "detector_type": "DLD",
+    "sensor_pixels": [
+      1800,
+      1800
+    ],
+    "amplifier_bias": "@attrs:metadata/file/KTOF:Lens:MCPfront:V",
+    "amplifier_bias/@units": "V",
+    "amplifier_voltage": "@attrs:metadata/file/KTOF:Lens:MCPback:V",
+    "amplifier_voltage/@units": "V",
+    "detector_voltage": "@attrs:metadata/file/KTOF:Lens:UDLD:V",
+    "detector_voltage/@units": "V"
+  },
+  "/ENTRY[entry]/INSTRUMENT[instrument]/source_TYPE[source_probe]": {
+    "name": "HHG @ TR-ARPES @ FHI",
+    "probe": "photon",
+    "type": "HHG laser",
+    "mode": "Single Bunch",
+    "frequency": "@attrs:metadata/instrument/beam/probe/frequency",
+    "frequency/@units": "kHz",
+    "photon_energy": "@attrs:metadata/instrument/beam/probe/incident_energy",
+    "photon_energy/@units": "eV",

```

Comment:
Well, if any this should be the energy _before_ conversion to XUV, i.e. the driving energy. I would rather remove this here (as we discussed already).